PulsePlayer/PulsePlayer/Assets.xcassets/Contents.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
What is the need of adding this new file? We already have one at below location:
https://github.com/INVIDITechnologies/pulse-sdk-ios-2.x-sample/blob/master/PulsePlayer/PulsePlayer/Assets.xcassets/AppIcon.appiconset/Contents.json